src/pages/Login/index.tsx

```diff
@@ -1,10 +1,15 @@
 import React from "react";
 
+// TODO: transfer below components to correct view
+import Table from "../../components/Table";
+import { useNetwork } from "../../network";
+
 const Login = () => {
+  const data = useNetwork();
 
   return (
     <div>
-      Login
+      <Table data={data}/>
```

Comment:
The table component will be in another view, e.g. Sessions, so you can create this view and move usage of Table there